[BUG]

### Describe the bug

The connection with server is terminated when users try to select something from add-on items (like insulator, light, train, car).

### To Reproduce

1. The connection ends unexpectedly while choosing something. It comes out more often when the model preview comes out.
2. After reconnecting, selecting something, especially clicking button, from most of add-on items causes connection lost.

### Expected behavior

Just smooth server play

### Actual Behavior

Connection lost continues to occur under descrived circumstances. Shutting down minecraft client nor PC didn't make thing better.

### Screenshots

Minecraft says
Connection Lost - A fatal error has occured, this connection is terminated.

### Crash Reports

It seems that Minecraft Client itself does not crash.

### OS

Windows Home 21H2

### KaizPatchX Version

1.4.6

### Other Mods

Mohist Bukkit

該当サーバーのlog(1)です。

[21:51:41 ERROR]: There was a critical exception handling a packet on channel RTM
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) ~[SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) ~[TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) ~[TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) ~[mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) ~[mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) ~[ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) ~[mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) ~[ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) ~[ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) ~[mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[21:51:41 INFO]: _Samhan lost connection: A fatal error has occured, this connection is terminated

### 何もしなかった時fatal Error logです。

`
[22:03:21 ERROR]: FMLIndexedMessageCodec exception caught
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: SimpleChannelHandlerWrapper exception
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 WARN]: More than one exception was raised. Will report only the first one and log others.
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) [SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) [TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) [TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) [mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) [mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) [mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) [ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) [ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) [mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 ERROR]: There was a critical exception handling a packet on channel RTM
io.netty.handler.codec.EncoderException: java.lang.NullPointerException
        at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107) ~[MessageToMessageEncoder.class:?]
        at io.netty.handler.codec.MessageToMessageCodec.write(MessageToMessageCodec.java:116) ~[MessageToMessageCodec.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.invokeWrite(DefaultChannelHandlerContext.java:644) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.write(DefaultChannelHandlerContext.java:698) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:688) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelHandlerContext.writeAndFlush(DefaultChannelHandlerContext.java:717) ~[DefaultChannelHandlerContext.class:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:893) ~[DefaultChannelPipeline.class:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:239) ~[AbstractChannel.class:?]
        at cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper.sendToAll(SimpleNetworkWrapper.java:182) ~[SimpleNetworkWrapper.class:1.7.10-45]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.sendPacket(TileEntityLargeRailCore.java:190) ~[TileEntityLargeRailCore.class:?]
        at jp.ngt.rtm.rail.TileEntityLargeRailCore.func_145844_m(TileEntityLargeRailCore.java:185) ~[TileEntityLargeRailCore.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_147097_b(EntityPlayerMP.java:721) ~[mw.class:?]
        at net.minecraft.entity.player.EntityPlayerMP.func_70071_h_(EntityPlayerMP.java:380) ~[mw.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2706) ~[ahb.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:878) ~[mt.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:2644) ~[ahb.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:2460) ~[ahb.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:674) ~[mt.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:437) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:835) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:692) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_321]
Caused by: java.lang.NullPointerException
[22:03:21 INFO]: _Samhan lost connection: A fatal error has occured, this connection is terminated
[22:03:21 INFO]: [ 루나 서버 ] _Samhan 님이 서버에서 퇴장하셨습니다! 내일 또 봐요!
[22:03:21 INFO]: _Samhan left the game.
`

--- a/src/main/java/jp/ngt/rtm/rail/TileEntityLargeRailCore.java
+++ b/src/main/java/jp/ngt/rtm/rail/TileEntityLargeRailCore.java
@@ -187,7 +187,9 @@ public abstract class TileEntityLargeRailCore extends TileEntityLargeRailBase {
     }
 
     public void sendPacket() {
-        RTMCore.NETWORK_WRAPPER.sendToAll(new PacketLargeRailCore(this, PacketLargeRailCore.TYPE_NORMAL));
+        if ((this.worldObj == null || !this.worldObj.isRemote) && this.isLoaded()) {
+            RTMCore.NETWORK_WRAPPER.sendToAll(new PacketLargeRailCore(this, PacketLargeRailCore.TYPE_NORMAL));
+        }
     }
 
     @Override

--- a/src/main/java/jp/ngt/rtm/rail/TileEntityLargeRailSlopeCore.java
+++ b/src/main/java/jp/ngt/rtm/rail/TileEntityLargeRailSlopeCore.java
@@ -45,7 +45,9 @@ public class TileEntityLargeRailSlopeCore extends TileEntityLargeRailCore {
 
     @Override
     public void sendPacket() {
-        RTMCore.NETWORK_WRAPPER.sendToAll(new PacketLargeRailCore(this, PacketLargeRailCore.TYPE_SLOPE));
+        if ((this.worldObj == null || !this.worldObj.isRemote) && this.isLoaded()) {
+            RTMCore.NETWORK_WRAPPER.sendToAll(new PacketLargeRailCore(this, PacketLargeRailCore.TYPE_SLOPE));
+        }
     }
 
     @Override

--- a/src/main/java/jp/ngt/rtm/rail/TileEntityLargeRailSwitchCore.java
+++ b/src/main/java/jp/ngt/rtm/rail/TileEntityLargeRailSwitchCore.java
@@ -102,7 +102,9 @@ public class TileEntityLargeRailSwitchCore extends TileEntityLargeRailCore {
 
     @Override
     public void sendPacket() {
-        RTMCore.NETWORK_WRAPPER.sendToAll(new PacketLargeRailCore(this, PacketLargeRailCore.TYPE_SWITCH));
+        if ((this.worldObj == null || !this.worldObj.isRemote) && this.isLoaded()) {
+            RTMCore.NETWORK_WRAPPER.sendToAll(new PacketLargeRailCore(this, PacketLargeRailCore.TYPE_SWITCH));
+        }
     }
 
     @Override

